public/gamecompare.php

```diff
@@ -189,8 +188,7 @@
                         echo GetAchievementAndTooltipDiv($achID, $achTitle, $achDesc, $achPoints, $gameTitle, $badgeName, true, true, "", $iconSize, "goldimage awardremote");
                         echo "</div>";
                         $rightAwardedCount++;
-                        $rightAwardedCount++;
-                        $leftAwardedPoints += $achPoints;
+                        $rightAwardedPoints += $achPoints;
```

Comment:
Why are we duplicating this line?

Reply:
Hardcore badges are worth double points. This is actually changing one instance of `leftAwardedPoints` to `rightAwardedPoints` as the hardcore points were being attributed to the wrong user. You can see the duplication for the left user on lines 159/160.

Reply:
That's a good point! Approved.